### PR TITLE
fix form nav button imports

### PR DESCRIPTION
**Pull request by amponce**
We noticed when participants were using Voiceovers on a Mac and go to the Streamlined Waiver transition page, it wouldn't set focus on the heading in the va-alert component.

Screenshot of fix:

Screenshot 2024-05-09 at 1 28 46 PM

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#69366

## Testing
Screen reader verification macbook pro m3

### CODE REVIEW

**Review by rmessina1010 — APPROVED** [review]
LGTM

File: src/applications/financial-status-report/components/resolution/ResolutionExplainerWidget.jsx [hunk 1]
```diff
@@ -41,9 +45,8 @@ const ResolutionExplainerWidget = ({
           close-btn-aria-label="Close notification"
           disable-analytics="false"
           full-width="false"
-          show-icon
```
Comment (Dr-Pongo) [inline src/applications/financial-status-report/components/resolution/ResolutionExplainerWidget.jsx:44]: Has `show-icon` been deprecated, or is it not necessary anymore?

Comment (amponce) [reply src/applications/financial-status-report/components/resolution/ResolutionExplainerWidget.jsx:44]: Yeah I'm pretty sure the icons are now controlled by the status prop. 

As far as I can tell show-icon was removed from the v3 and the V1 component props

https://design.va.gov/storybook/?path=/docs/uswds-va-alert

**Review by josephrlee — COMMENTED** [review]
Nice work!

**Pull request merged**